docs/cloud/automation/README.md

```diff
@@ -1,3 +1,18 @@
 # Cloud automation
 
-In addition to the Cloud console, EventStoreDB Cloud provides the API and the Terraform provider to automate any operation accessible from the console.
+In addition to the Cloud console, EventStoreDB Cloud provides the API and a Terraform provider to automate any operation accessible from the console.
+
+## Contributing
+
+The Event Store Cloud Terraform provider is released under the [Mozilla Public License version 2][mpl 2.0], like most Terraform
+providers.  
+We welcome pull requests and issues!  
+We adhere to the [Contributor Covenant][codeofconduct] Code of Conduct,
```

Comment:
This contributor guidelines need checking and agreeing on

Reply:
it's in the readme in the repo as well :
https://github.com/EventStore/terraform-provider-eventstorecloud#contributing
I can just remove that ATM